Cargo.toml

```diff
@@ -27,6 +27,9 @@ bitflags = "0.9"
 cfg-if = "0.1.0"
 void = "1.0.2"
 
+[target.'cfg(target_os = "dragonfly")'.dependencies]
+errno-dragonfly = "0.1"
```

Comment:
We shouldn't need an extra crate just for this.  The basic symbols should all be defined in libc, and nix's won src/errno.rs defines `errno_location`.  Can you merge whatever you need into nix and libc so we don't depend on the extra crate?

Reply:
I don't have a problem with this crate staying external, though I'd much rather rely on a crate that had proper semver (i.e. was 1.0+).